app/views/events/aggregate.html.erb

```diff
@@ -1,14 +1,19 @@
 <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/vis/4.21.0/vis.min.js"></script>
 <link type="text/css" src="https://cdnjs.cloudflare.com/ajax/libs/vis/4.21.0/vis.min.css" />
 <script src="https://cdnjs.cloudflare.com/ajax/libs/pusher/4.3.1/pusher.min.js"></script>
-
 <script type="text/javascript" src="/assets/event-stream.js"></script>
 <script type="text/javascript" src="/assets/event-visualization.js"></script>
-<script type="text/javascript" src="/assets/event-creation-app.js"></script>
+
+<h1>Aggregate Events</h1>
 
 <div id='mynetwork' style="height:300px; width: 800px; border: 1px solid black;"></div>
+
 <script type="text/javascript">
-  const diagram = createBlankNetworkDiagram('mynetwork');
-  retrieveAllEvents().then(events => events.forEach(event => addEventInstanceAsAggregateNodes(diagram, event)));
-  listenForPusherEvents(event => addEventInstanceAsAggregateNodes(diagram, event));
```

Comment:
@calebcauthon Look at the ```events => events.forEach(event => addEventInstanceAsAggregateNodes(diagram, event))``` code here.  It should look familiar, as it was already in here:  https://github.com/darrencauthon/fasten/pull/53/files#diff-cd4ecb39b98c53b31ff9e2a12af30838L16 .... but in a method called "show event bubbles."

I get that method and I don't think "show event bubbles" was a bad name when it was written.  It wasn't till I started looking at this code and started a few refactors that the picture snapped into place.  You can add multiple events to a diagram.  You can add a single event to a diagram.  So why not have both function the same way?